platform/solr/solr-factory-impl/src/main/java/org/codice/solr/factory/impl/SolrClientAdapter.java

```diff
@@ -446,7 +446,15 @@ public void whenAvailable(Initializer initializer) {
   // which goes throw getProxiedClient() which would throw back an unavailable error instead of
   // returning the response
   public SolrPingResponse ping() throws SolrServerException, IOException {
-    return ping("from the API");
+    return ping(true, null);
+  }
+
+  @Override
+  // overridden to always send the ping to the client; avoiding the intercept in handle()
+  // which goes throw getProxiedClient() which would throw back an unavailable error instead of
```

Comment:
✏️  Did you mean `which goes through geteProxiedClient()` instead?

Reply:
The comment was copied and pasted from the method above.  Looks like that was missed in the original review.